Common/include/option_structure.hpp

```diff
@@ -335,7 +335,35 @@ static const MapType<string, ENUM_RADIALBASIS> RadialBasisFunction_Map = {
 };
 
 /*!
- * \brief Types of (coupling) transfers between distinct physical zones
+ * \brief type of radial spanwise interpolation function for the inlet face
+ */
+enum ENUM_INLET_SPANWISEINTERPOLATION {
+  NO_INTERPOLATION = 0,
+  LINEAR_1D = 1,
+  AKIMA_1D = 2,
+};
+static const map<string, ENUM_INLET_SPANWISEINTERPOLATION> Inlet_SpanwiseInterpolation_Map = {
+  MakePair("NONE", NO_INTERPOLATION)
+  MakePair("LINEAR_1D",LINEAR_1D)
+  MakePair("AKIMA_1D",AKIMA_1D)
+};
+
+/*!
+ * \brief type of radial spanwise interpolation data type for the inlet face
+ */
+enum ENUM_INLET_INTERPOLATIONTYPE {
+  VR_VTHETA = 0,
+  ALPHA_PHI = 1,
+  BETA_PHI = 2,
+};
+static const map<string, ENUM_INLET_INTERPOLATIONTYPE> Inlet_SpanwiseInterpolationType_Map = {
+  MakePair("VR_VTHETA",VR_VTHETA)
+  MakePair("ALPHA_PHI",ALPHA_PHI)
+  MakePair("BETA_PHI",BETA_PHI)
```

Comment:
Yes. Please see the link I have set in the text of the pull request above. I can post it here again. 
[Link to files](https://mailuc-my.sharepoint.com/:f:/g/personal/baigan_mail_uc_edu/Ep03wpFrJW1Hvwa73U_Sp6QBaZ_dNyyynRcwWZFCIMF0lQ?e=daN2r1).
You can see the files for velocity triangles. There it has all the equations and terminologies I that I have used, I have also highlighted them. Including alpha, phi, Vr, VTheta etc.
The reason I use to update everything on my fork is that I am using one machine to edit and another machine to run my code through remote. I don't want regression tests to be kicking off every time I push something here.

Reply:
Edit your first post and attach the files directly, give a brief description of the options and reference this pull request in your example testcase, otherwise no one will find the files.

Reply:
done